decidim-proposals/app/views/decidim/proposals/proposals/preview.html.erb

```diff
@@ -1,41 +1,73 @@
 <% add_decidim_page_title(proposal_wizard_step_title(action_name)) %>
 
-<div class="row">
-  <%= render partial: "wizard_aside" %>
-
-  <div class="columns large-9">
-    <%= render partial: "wizard_header", locals: { callout_help_text_class: "warning" } %>
-    <div class="card card__content">
-      <div class="row column view-header">
-        <h3 class="heading3"><%= present(@proposal).title(links: true, html_escape: true) %></h3>
-        <% unless component_settings.participatory_texts_enabled? %>
-          <%= cell("decidim/coauthorships", @proposal, has_actions: false, size: 3, context: { current_user: current_user }) %>
-        <% end %>
+<% content_for :css_content do %>
+  <%= stylesheet_pack_tag "decidim_proposals", media: "all" %>
+<% end %>
 
-        <%= render_proposal_body(@proposal) %>
-      </div>
-      <div class="row column">
-        <%= cell "decidim/proposals/proposal_tags", @proposal %>
-      </div>
-    </div>
-    <% if component_settings.geocoding_enabled? %>
-      <% if has_position?(@proposal) %>
-      <div class="row column">
-        <h3 class="section-heading"><%= t("decidim.application.geocoding.geocoded_address") %></h3>
-            <%= render partial: "decidim/shared/static_map", locals: { icon_name: "proposals", geolocalizable: @proposal } %>
-      </div>
-      <% end %>
+<% content_for :js_content do %>
+  <%= javascript_pack_tag "decidim_proposals" %>
+<% end %>
+
+<%= render layout: "layouts/decidim/shared/layout_center" do %>
+
+  <%= render partial: "wizard_header", locals: { callout_help_text_class: "warning" } %>
+
+  <div class="proposal__container my-10">
+    <h2 class="h3 text-secondary"><%= present(@proposal).title(links: true, html_escape: true) %></h2>
+
+    <% unless component_settings.participatory_texts_enabled? %>
+      <%#= cell("decidim/coauthorships", @proposal, has_actions: false, size: 3, context: { current_user: current_user }) %>
```

Comment:
do we need this comment?

Reply:
Not really, the redesign of coauthorships is redesign pending, I've keep the `REDESIGN_PENDING` comment line